core/include/detray/utils/consistency_checker.hpp

```diff
@@ -252,9 +252,7 @@ inline void check_empty(const detector_t &det, const bool verbose) {
     // Warnings
 
     // Check the material description
-    if (det.material_store().all_empty()) {
```

Comment:
Please leave this in, it is useful in IO debugging and when combining a geometry file with new/old/different material files. If it bothers you, you can do ```tel_config.do_check(false)``` once your code is production ready

Reply:
Ok, I am not sure why that would happen. Can you print the size of the volume material collection above the warning?
Something like 
```
using material_id = typename detector_t::materials::id;
det.material_store().template size<static_cast<material_id>(0)>()
```